css/index.css

```diff
@@ -104,6 +133,7 @@ p {
 .middle-content .boxes .box {
     width: 12.5%;
     height: 100px;
+    font-size: 1.5rem;
```

Comment:
adding a `min-width` to these boxes would have stopped them from squishing on browser resize.